code/datums/components/material_container.dm

```diff
@@ -21,14 +21,18 @@
 	var/precise_insertion = FALSE
 	var/datum/callback/precondition
 	var/datum/callback/after_insert
+	var/list/blacklist
 
 /// Sets up the proper signals and fills the list of materials with the appropriate references.
-/datum/component/material_container/Initialize(list/mat_list, max_amt = 0, _show_on_examine = FALSE, list/allowed_types, datum/callback/_precondition, datum/callback/_after_insert, _disable_attackby)
+/datum/component/material_container/Initialize(list/mat_list, max_amt = 0, _show_on_examine = FALSE, list/allowed_types, list/_blacklisted_types, datum/callback/_precondition, datum/callback/_after_insert, _disable_attackby)
```

Comment:
simply because of this I'd like this to be testmerged for a while because we all know how shit bugs caused by these types of changes can be

Reply:
uh yes i was actually about to delete it today coz i found a better solution for what im looking for